htmd/builder/charmmfiles/par/par_all22_prot.prm

```diff
@@ -35,53 +35,53 @@
 !
 
 ATOMS
-MASS    41 H      1.00800 ! polar H
-MASS    42 HC     1.00800 ! N-ter H
-MASS    43 HA     1.00800 ! nonpolar H
-MASS    44 HP     1.00800 ! aromatic H
-MASS    45 HB     1.00800 ! backbone H
-MASS    46 HR1    1.00800 ! his he1, (+) his HG,HD2
-MASS    47 HR2    1.00800 ! (+) his HE1
-MASS    48 HR3    1.00800 ! neutral his HG, HD2
-MASS    49 HS     1.00800 ! thiol hydrogen
-MASS    50 HE1    1.00800 ! for alkene; RHC=CR
-MASS    51 HE2    1.00800 ! for alkene; H2C=CR
-MASS    52 C     12.01100 ! carbonyl C, peptide backbone
-MASS    53 CA    12.01100 ! aromatic C
-MASS    54 CT1   12.01100 ! aliphatic sp3 C for CH
-MASS    55 CT2   12.01100 ! aliphatic sp3 C for CH2
-MASS    56 CT3   12.01100 ! aliphatic sp3 C for CH3
-MASS    57 CPH1  12.01100 ! his CG and CD2 carbons
-MASS    58 CPH2  12.01100 ! his CE1 carbon
-MASS    59 CPT   12.01100 ! trp C between rings
-MASS    60 CY    12.01100 ! TRP C in pyrrole ring
-MASS    61 CP1   12.01100 ! tetrahedral C (proline CA)
-MASS    62 CP2   12.01100 ! tetrahedral C (proline CB/CG)
-MASS    63 CP3   12.01100 ! tetrahedral C (proline CD)
-MASS    64 CC    12.01100 ! carbonyl C, asn,asp,gln,glu,cter,ct2
-MASS    65 CD    12.01100 ! carbonyl C, pres aspp,glup,ct1
-MASS    66 CS    12.01100 ! thiolate carbon
-MASS    67 CE1   12.01100 ! for alkene; RHC=CR
-MASS    68 CE2   12.01100 ! for alkene; H2C=CR
-MASS    69 C3    12.01100 ! cyclopropyl carbon
-MASS    70 N     14.00700 ! proline N
-MASS    71 NR1   14.00700 ! neutral his protonated ring nitrogen
-MASS    72 NR2   14.00700 ! neutral his unprotonated ring nitrogen
-MASS    73 NR3   14.00700 ! charged his ring nitrogen
-MASS    74 NH1   14.00700 ! peptide nitrogen
-MASS    75 NH2   14.00700 ! amide nitrogen
-MASS    76 NH3   14.00700 ! ammonium nitrogen
-MASS    77 NC2   14.00700 ! guanidinium nitroogen
-MASS    78 NY    14.00700 ! TRP N in pyrrole ring
-MASS    79 NP    14.00700 ! Proline ring NH2+ (N-terminal)
-MASS    80 O     15.99900 ! carbonyl oxygen
-MASS    81 OB    15.99900 ! carbonyl oxygen in acetic acid
-MASS    82 OC    15.99900 ! carboxylate oxygen
-MASS    83 OH1   15.99900 ! hydroxyl oxygen
-MASS    84 OS    15.99940 ! ester oxygen
-MASS    85 S     32.06000 ! sulphur
-MASS    86 SM    32.06000 ! sulfur C-S-S-C type
-MASS    87 SS    32.06000 ! thiolate sulfur
+MASS  -1  H          1.00800 ! polar H
```

Comment:
This is a nightmare to review, as I have no way to know if this is correct. I strongly recommend we should decouple the charmm ff from the htmd package. And the way to create these files should be documented or scripted (like I attempted to do in here: https://github.com/Acellera/conda-charmmff/blob/master/charmmff/Makefile). Otherwise, mistakes will be made and we won't have a way to know.

Reply:
I agree. But what am I supposed to do now?

Reply:
Merge without review.